scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1588,6 +1588,12 @@ If ([string]::IsNullOrWhiteSpace($MetricsUploadTarget))
     $MetricsScrapeFrequencyInSecs="AzureLogAnalytics";
 }
 
+If ($TestName -eq "LongHaul" -Or $TestName -eq "Stress")
```

Comment:
I followed the existing convention, however we use double comma syntax for our linux shell script in order to check the lower cased string, so we aren't being consistent.